README.md

```diff
@@ -6,17 +6,16 @@
 # Compiler Explorer
 
 **Compiler Explorer** is an interactive compiler exploration website. Edit code in C, C++, C#, F#, Rust, Go, D, Haskell,
-Swift, Pascal, [ispc](https://ispc.github.io/), Python, Java or in any of the other
-[30+ supported languages](https://godbolt.org/api/languages), and see how that code looks after being compiled in real
-time. Multiple compilers are supported for each language, many different tools and visualisations are available, and the
+Swift, Pascal, [ispc](https://ispc.github.io/), Python, Java, or any of the other
+[30+ supported languages](https://godbolt.org/api/languages), and see how that code looks after being compiled in real time. Multiple compilers are supported for each language, many different tools and visualizations are available, and the
```

Comment:
Can you keep to the same line limit as the other lines here. I thought we had a pre-commit hook for this...sorry it didn't report anything.